src/renderer/vt/tracing.cpp

```diff
@@ -43,7 +43,7 @@ std::string toPrintableString(const std::string_view& inString)
     for (size_t i = 0; i < inString.length(); i++)
     {
         unsigned char c = inString[i];
-        if (c < '\x20' && c < '\x7f')

```

Comment:
Actually, for this one, the `&& c < '\x7f'` clause should probably just be removed - the `c < '\x20' ` part was the important part.

Reply:
If that is the case, then the else if (c == '\x7f') block can also be removed

Reply:
Perhaps `&& c != '\x7f'` instead?

Reply:
Perhaps `&& c != '\x7f'` instead?

Reply:
woops I was looking at `c > '\x20` the whole time.